policy/README.md

```diff
@@ -26,22 +27,43 @@ Additionally, you need to specify the service account credentials as an environm
 
 ## Add a policy to Mondoo Platform
 
-You can use the Action as follows:
+The following example runs `cnspec bundle validate` on specified policies on all pull requests. Once a PR is approved and merged the workflow will run `cnspec bundle publish` on specified policies.
 
 ```yaml
 name: Mondoo Policy Add Example
 on:
   push:
     paths:
-      - "policy/policy.yml"
+      - "policy/policy.mql.yaml"
+
 jobs:
-  upload:
+  cnspec-policy-validate:
+    name: Validate cnspec policy bundle
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: mondoohq/actions/policy@scottford/updates-policy-action
+        env:
+          MONDOO_CONFIG_BASE64: ${{ secrets.MONDOO_SERVICE_ACCOUNT }}
+        with:
+          action: validate
```

Comment:
Isn't that conflicting with https://github.com/mondoohq/actions/tree/main/cnspec-lint? We also renamed it to lint